website/docs/r/dx_connection_association.html.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: "aws"
+page_title: "AWS: aws_dx_connection_association"
+sidebar_current: "docs-aws-resource-dx-connection-association"
+description: |-
+  Associates a Connection with LAG.
+---
+
+# aws_dx_connection_association
+
+Associates a Connection with LAG.
```

Comment:
Nitpick, but do you mind changing this to `Associates a Direct Connect Connection with a LAG.`?